docs/maui/native-library-interop/index.md

```diff
@@ -0,0 +1,86 @@
+
+A key benefit of Native Library Interop is its effectiveness with simple API surfaces. When wrappers involve only primitive types that .NET supports, existing binding tools can generate reliable definitions with minimal manual intervention, which is often required for traditional bindings. This makes the process straightforward, especially since wrapper API implementation typically follows SDK documentation, and often allows direct copying from vendor documentation.
+
+While the initial setup may be detailed, managing updates to underlying SDKs generally requires less effort. Updates often involve simply adjusting the version and rebuilding the project. Even if breaking changes occur in the API surfaces or SDKs, the wrapper API surface and .NET application's usage are likely to remain stable, requiring fewer adjustments compared to traditional bindings.
+
+In summary, Native Library Interop provides several benefits:
+- Simplifies following SDK documentation with native languages and tools
+- Requires minimal manual intervention to create working bindings
+- Makes maintenance easier and reduces the frequency of necessary updates
+- Enhances isolation of the app from changes in underlying SDKs
+
+Although resolving dependency chains (particularly on Android) may require similar effort as traditional bindings, the streamlined implementation and maintenance advantages make Native Library Interop an appealing choice for many projects.
+
+
+### Understanding Maui.NativeLibraryInterop
+
+A notable challenge with creating and maintaining bindings created via Native Library Interop is manually coalescing the native projects, their native dependencies, build outputs, and the .NET Binding library project. Maui.NativeLibraryInterop helps you jumpstart the process by building from and customizing the samples for your own app's needs.
+
+Part of this includes orchestrating parts of the build process through MSBuild invocations. This can include:
+
+- Resolving or downloading native SDK dependencies
+- Building the native slim binding project and its dependencies
+- Moving the requisite native artifacts to the expected working directory
+- Generating the API definition for the binding library project
+
+The binding build process is extended to obtain and build native SDK dependencies by adding the `CommunityToolkit.Maui.NativeLibraryInterop.BuildTasks` NuGet package to your binding project:
+
+```xml
+  <ItemGroup>
+    <PackageReference Include="CommunityToolkit.Maui.BindingExtensions" Version="0.0.1-pre1" />
+  </ItemGroup>
+```
+
+Android binding projects will also add a `@(GradleProjectReference)` item that points to the root folder that contains their gradle project:
+
+```xml
+<ItemGroup>
+    <GradleProjectReference Include="../native" >
+        <ModuleName>{module_name}</ModuleName>
+        <!-- Metadata applicable to @(AndroidLibrary) will be used if set -->
+        <Bind>true</Bind>
+        <Pack>true</Pack>
```

Comment:
I think we can comment these out to show that they are optional with the following default values:
```suggestion
       <!-- Metadata applicable to @(AndroidLibrary) will be used if set, otherwise the following defaults will be used:
      <Bind>true</Bind>
      <Pack>true</Pack>
      -->
```